website/docs/quick-start.md

```diff
@@ -31,32 +31,37 @@ This sample illustrates how to patch containers using vulnerability reports with
 
-3. Patch the image using the Trivy report. You will need to start `buildkitd` if it is not already running:
-
+3. Patch the image using the Trivy report.:
+
+    You'll need a buildkit instance to connect to.
+    By default copa will attempt to auto-connect to an instance in order:
+        1. Default docker buildkit endpoint
+        2. Currently selected buildx builder (see: `docker buildx --help`)
+        3. buildkit daemon at the default address `/run/buildkit/buildkitd.sock`
+    
+    If an instance doesn't exist or that instance doesn't support all the features copa needs the next will be attempted.
+    You may need to specify a custom address using the `--addr` flag. Here are the supported formats:
+
+    - `unix:///path/to/buildkit.sock` - Connect to buildkit over unix socket.
+    - `tcp://$BUILDKIT_ADDR:$PORT` - Connec to buildkit over TCP. (not recommended for security reasons)
+    - `docker://<docker connection spec>` - Connect to docker, currently only unix sockets are supported, e.g. `docker://unix:///var/run/docker.sock` (or just `docker://`).
+    - `docker-container://my-buildkit-container` - Connect to a buildkitd running in a docker container.
+    - `buildx://my-builder` - Connect to a buildx builder (or `buildx://` for the currently selected builder). *Note: only container-backed buildx instances are currently supported*
+    - `nerdctl-container://my-container-name` - Similar to `docker-container` but uses `nerdctl`.
+    - `podman-container://my-container-name` - Similar to `docker-container` but uses `podman`.
+    - `ssh://myhost` - Connect to a buildkit instance over SSH. Format of the host spec should mimic the SSH command.
+    - `kubepod://mypod` - Connect to buildkit running in a Kubernetes pod. Can also specify kubectl context and pod namespace (`kubepod://mypod?context=foo&namespace=notdefault`).
+
+
+    Example: Connect using defaults: 
     ```bash
-    sudo buildkitd &
-    sudo copa patch -i mcr.microsoft.com/oss/nginx/nginx:1.21.6 -r nginx.1.21.6.json -t 1.21.6-patched
+    copa patch -i mcr.microsoft.com/oss/nginx/nginx:1.21.6 -r nginx.1.21.6.json -t 1.21.6-patched
     ```
 
-    Alternatively, you can run `buildkitd` in a container, which allows copa to be run without root access to the local buildkit socket:
-
-    ```bash
-    export BUILDKIT_VERSION=v0.12.0
-    export BUILDKIT_PORT=8888
-    docker run \
-        --detach \
-        --rm \
-        --privileged \
-        -p 127.0.0.1:$BUILDKIT_PORT:$BUILDKIT_PORT/tcp \
-        --name buildkitd \
-        --entrypoint buildkitd \
-        "moby/buildkit:$BUILDKIT_VERSION" \
-        --addr tcp://0.0.0.0:$BUILDKIT_PORT
-    copa patch \
-        -i mcr.microsoft.com/oss/nginx/nginx:1.21.6 \
-        -r nginx.1.21.6.json \
-        -t 1.21.6-patched \
-        -a tcp://0.0.0.0:$BUILDKIT_PORT
+    Example: Connect to buildx
+    ```
+    docker buildx create --name demo
```

Comment:
do we need --use? Will this bootstrap automatically?

Reply:
Didn't want to do `--use` so as to show we are using the named instance.
The driver does bootstrap the builder automatically.